resource-timing/resources/all_resource_types.htm

```diff
@@ -44,6 +67,32 @@
         <embed src="resource_timing_test0.css?id=embed" type="text/css"></embed>
         <input type="image" src="blue.png?id=input"></input>
         <object type="image/png" data="blue.png?id=object"></object>
-        <video poster="blue.png?id=poster"></video>
+        <video poster="blue.png?id=video-poster"></video>
+        <video src="/media/test.mp4?id=video-src" autoplay="true"></video>
+        <video autoplay="true">
+            <source src="/media/test.mp4?id=video-source" type="video/mp4">
+            <source src="/media/test.ogv?id=video-source" type="video/ogg">
+            <track kind="subtitles" src="empty.py?id=video-track" srclang="en" default>
+        </video>
+        <audio src="empty.py?id=audio-src"></audio>
+        <audio>
+            <source src="empty.py?id=audio-source-wav" type="audio/wav" />
+            <source src="empty.py?id=audio-source-mpeg" type="audio/mpeg" />
+            <source src="empty.py?id=audio-source-ogg" type="audio/ogg" />
+        </audio>
+        <svg width=200 height=200
+            xmlns="http://www.w3.org/2000/svg"
+            xmlns:xlink="http://www.w3.org/1999/xlink">
+            <image href="blue.png?id=svg-image" height="200" width="200"/>
+        </svg>
+        <picture>
+            <source srcset="blue.png?id=picture-source" type="image/png" />
+            <img src="blue.png?id=picture-img" />
```

Comment:
`picture-img` won't normally download here, so that part of the test won't actually run anywhere other than legacy browsers. Might be worth while to add another test case where the source's media attribute doesn't match.

Reply:
Great idea, added via d6079f5dcbc8595d2be4d495d46860172e597ea3